codegen/templates/deployers/HyperdriveDeployerCoordinator.sol.jinja

```diff
@@ -135,6 +135,24 @@ contract {{ name.capitalized }}HyperdriveDeployerCoordinator is HyperdriveDeploy
         return value;
     }
 
+    /// @notice Convert an amount of vault shares to an amount of base.
+    /// @param _shareAmount The vault shares amount.
+    /// @return The base amount.
+    function convertToBase(
+        uint256 _shareAmount
+    ) public view returns (uint256) {
+        // FIXME
```

Comment:
unresolved fixme

Reply:
This is a template. It’s meant to be there